src/Symfony/Component/Notifier/Bridge/Mercure/MercureOptions.php

```diff
@@ -22,6 +22,22 @@ final class MercureOptions implements MessageOptionsInterface
 
     /**
      * @param string|string[]|null $topics
+     * @param array{
+     *     actions?: array,
+     *     badge?: string,
+     *     body?: string,
+     *     data?: mixed,
+     *     dir?: 'auto'|'ltr'|'rtl',
+     *     icon?: string,
+     *     image?: string,
+     *     lang?: string,
+     *     renotify?: bool,
+     *     requireInteraction?: bool,
+     *     silent?: bool,
+     *     tag?: string,
+     *     timestamp?: int,
+     *     vibrate?: int[],
```

Comment:
should be `list<int>` (if it is not a list, it won't be an Array in JS as JS Array is not the equivalent of a PHP array but is strictly a list)

Reply:
btw, according to https://developer.mozilla.org/en-US/docs/Web/API/Vibration_API#concepts_and_usage, it should probably be `int|list<int>` as passing a single number seems to be supported.